src/ui/TemplateMessageItemBody/index.tsx

```diff
@@ -242,11 +242,15 @@ export function TemplateMessageItemBody({
               throw new Error('TemplateMessageItemBody | template key suggests composite template but template data is missing view_variables. See error log in console for details');
             }
             const reservationKey = removeAtAndBraces(carouselItem.items);
-            const simpleTemplateDataList: SimpleTemplateData[] | undefined = templateData.view_variables[reservationKey];
+            let simpleTemplateDataList: SimpleTemplateData[] | undefined = templateData.view_variables[reservationKey];
             if (!simpleTemplateDataList) {
               logger.error('TemplateMessageItemBody | no reservation key found in view_variables: ', reservationKey, templateData.view_variables);
               throw new Error('TemplateMessageItemBody | no reservation key found in view_variables. See error log in console for details');
             }
+            if (simpleTemplateDataList.length > 10) {
+              logger.warning('TemplateMessageItemBody | composite template with more than ten simple templates will only render the first ten items: ', reservationKey, templateData.view_variables);
```

Comment:
better for readability?
```suggestion
              logger.warning('TemplateMessageItemBody | composite template with more than 10 simple templates will only render the first 10 items: ', reservationKey, templateData.view_variables);
```